src/Microsoft.OpenApi/Services/OpenApiWorkspace.cs

```diff
@@ -62,7 +39,7 @@ public OpenApiWorkspace(Uri baseUrl)
         /// </summary>
         public OpenApiWorkspace()
         {
-            BaseUrl = new("file://" + Environment.CurrentDirectory + $"{Path.DirectorySeparatorChar}" );
+            BaseUrl = new Uri(OpenApiConstants.BaseRegistryUri);
```

Comment:
I'm fighting with the feeling that "it can't be this simple", but if every document gets it's own GUID in the baseURI then we really don't need to use a file path.  I _think_ this is really good.